api/v1alpha1/globalratelimit_types.go

```diff
@@ -56,6 +56,8 @@ type GlobalRateLimit_Action_RequestHeaders struct {
 	// value is used to populate the value of the descriptor entry for the
 	// descriptor_key.
 	HeaderName string `json:"header_name,omitempty" yaml:"header_name,omitempty"`
+	// The value to use in the descriptor entry.
+	DescriptorValue string `json:"descriptor_value,omitempty" yaml:"descriptor_value,omitempty"`
```

Comment:
let's make it as pointer so it's not required
```
DescriptorValue *string `json:"descriptor_value,omitempty" yaml:"descriptor_value,omitempty"`
```

Reply:
Good catch. Thanks